src/random_test.cpp

```diff
@@ -1607,20 +1609,74 @@ void Table::DeleteRandomRow(Thd1 *thd) {
       }
     }
   }
```

Comment:
small suggestion to reduce the size of code.
 start with sql = "DELETE FROM " + name_ + " WHERE " + columns_->at(where)->name_ in the first line and then append.

Reply:
Done